README.md

```diff
@@ -17,7 +17,22 @@ Installation guide for:
   - [Linux](https://radar-software-toolkit-rst.readthedocs.io/en/latest/user_guide/linux_install/)
   - [MacOSX](https://radar-software-toolkit-rst.readthedocs.io/en/latest/user_guide/mac_install/)
   - <font color="grey">Windows </font> yet to be implemented
-  
+
+## Citing SuperDARN Data 
+The Super Dual Auroral Radar Network (SuperDARN) is a made up of nominally 36 radars and 20 institutions, to cite general SuperDARN you can use:
+
+Greenwald, R.A., Baker, K.B., Dudeney, J.R. et al. Space Sci Rev (1995) 71: 761. doi:10.1007/BF00751350
+
+For the general achievements of the SuperDARN Network, you can read these papers:
+
+Chisham, G., Lester, M., Milan, S.E. et al. A decade of the Super Dual Auroral Radar Network (SuperDARN): scientific achievements, new techniques and future directions. Surv Geophys 28, 33–109 (2007) doi:10.1007/s10712-007-9017-8
+
+Nishitani, N., Ruohoniemi, J.M., Lester, M. et al. Review of the accomplishments of mid-latitude Super Dual Auroral Radar Network (SuperDARN) HF radars. Prog Earth Planet Sci 6, 27 (2019) doi:10.1186/s40645-019-0270-5
```

Comment:
I know this is copied straight from the SuperDARN Canada website, but it could be made slightly more future-proof regarding the number of radars. It's also common practice to cite all three review papers in publications, so I don't think that Greenwald et al., (1995) needs to be listed separately to the others:

```suggestion
## Citing SuperDARN Data 
The Super Dual Auroral Radar Network (SuperDARN) consists of more than 30 radars operated by institutions in 10 different countries. The following review papers are recommended for general information about the network's scientific achievements, and for citing SuperDARN in publications:

Greenwald, R.A., Baker, K.B., Dudeney, J.R. et al. Space Sci Rev (1995) 71: 761. doi:10.1007/BF00751350

Chisham, G., Lester, M., Milan, S.E. et al. A decade of the Super Dual Auroral Radar Network (SuperDARN): scientific achievements, new techniques and future directions. Surv Geophys 28, 33–109 (2007) doi:10.1007/s10712-007-9017-8

Nishitani, N., Ruohoniemi, J.M., Lester, M. et al. Review of the accomplishments of mid-latitude Super Dual Auroral Radar Network (SuperDARN) HF radars. Prog Earth Planet Sci 6, 27 (2019) doi:10.1186/s40645-019-0270-5
```

Reply:
agreed with @ecbland, although the Nishitani et al [2019] reference is primarily focused on mid-latitude SuperDARN results, so if one were using only high- or polar radar data it may not be necessary to cite.